src/scrabble.js

```diff
@@ -1,43 +1,69 @@
+  constructor(word) {
     this.word = word;
   }
-  score () { //Function in the class is a method. This is a method.
-    let score = 0 //Flexible variable
-  if (typeof this.word !== "string") { //If it's NOT a string, return 0. This is a conditional flow with a strict inequality (!==)
-    return 0;
-  } else {
-  const wordArray = this.word.toUpperCase().split("") // variable with array capitalising the splitted word.
-  for (let i = 0; i <= wordArray.length; i++) { 
-    if (wordArray[i] === "A" || wordArray[i] === "E" || wordArray[i] === "I" || wordArray[i] === "O" || wordArray[i] === "U" || wordArray[i] === "L" || wordArray[i] === "N" || wordArray[i] === "R" || wordArray[i] === "S" || wordArray[i] === "T") {
-  score += 1;
-  } else if (wordArray[i] === "D" || wordArray[i] === "G") {
-  score += 2;
-  } else if (wordArray[i] === "B" || wordArray[i] === "C" || wordArray[i] === "M" || wordArray[i] === "P") {
-  score += 3;
-  } else if (wordArray[i] === "F" || wordArray[i] === "H" || wordArray[i] === "V" || wordArray[i] === "W" || wordArray[i] === "Y" ) {
-  score += 4;
-  } else if (wordArray[i] === "K") {
-  score += 5;
-  } else if (wordArray[i] === "J" || wordArray[i] === "X") {
-  score += 8;
-  } else if (wordArray[i] === "Q" || wordArray[i] === "Z") {
-    score += 10
-  }
-}
-  }
+  score() {
+    //Function in the class is a method. This is a method.
+    let score = 0; //Flexible variable
+    if (typeof this.word !== "string") {
+      //If it's NOT a string, return 0. This is a conditional flow with a strict inequality (!==)
+      return 0;
+    } else {
+      const wordArray = this.word.toUpperCase().split(""); // variable with array capitalising the splitted word.
+      console.log(wordArray);
+      for (let i = 0; i <= wordArray.length; i++) {
+        if (
+          wordArray[i] === "A" ||
+          wordArray[i] === "E" ||
+          wordArray[i] === "I" ||
+          wordArray[i] === "O" ||
+          wordArray[i] === "U" ||
+          wordArray[i] === "L" ||
+          wordArray[i] === "N" ||
+          wordArray[i] === "R" ||
+          wordArray[i] === "S" ||
+          wordArray[i] === "T"
+        ) {
+          score += 1;
+        } else if (wordArray[i] === "D" || wordArray[i] === "G") {
+          score += 2;
+        } else if (
+          wordArray[i] === "B" ||
+          wordArray[i] === "C" ||
+          wordArray[i] === "M" ||
+          wordArray[i] === "P"
+        ) {
+          score += 3;
+        } else if (
+          wordArray[i] === "F" ||
+          wordArray[i] === "H" ||
+          wordArray[i] === "V" ||
+          wordArray[i] === "W" ||
+          wordArray[i] === "Y"
+        ) {
+          score += 4;
+        } else if (wordArray[i] === "K") {
+          score += 5;
+        } else if (wordArray[i] === "J" || wordArray[i] === "X") {
+          score += 8;
+        } else if (wordArray[i] === "Q" || wordArray[i] === "Z") {
+          score += 10;
+        }
+      }
+    }
```

Comment:
Nice way of doing it, it's really an intriguing way. However, in point-system try to use objects and arrays for letter scores rather than using conditional-flow because you would lose modularity and adding extra values would be difficult. Other than that it wasn't deal breaker since we're all learning and finding different ways of doing it will make growth exponentially faster.